onnxruntime/test/providers/cpu/activation/activation_op_test.cc

```diff
@@ -32,6 +33,18 @@ void TestUnaryElementwiseOp(const char* szOp, std::vector<float>& input_vals,
   if (!is_tensorrt_supported) {
     excluded_providers.insert(kTensorrtExecutionProvider);
   }
+
+//Disabled because of accuracy issues for MYRIAD FP16 and VAD_R
+#if defined(OPENVINO_CONFIG_MYRIAD) || defined(OPENVINO_CONFIG_VAD_R)
```

Comment:
would it be easier to use the same technique as the element_wise_op_test ?
looks like the changes here is causing a build warning/failure on Windows